masterdata/returnRequest/schema.json

```diff
@@ -0,0 +1,211 @@
+{
+  "properties": {
+    "orderId": {
+      "type": "string",
+      "required": true
+    },
+    "totalReturnAmount": {
+      "type": "integer",
+      "required": true
+    },
+    "sequenceNumber": {
+      "type": "string",
+      "required": true
+    },
+    "status": { "$ref": "#/$defs/status", "required": true },
```

Comment:
The `status` field is here to make it easy to SEARCH for order status. There is an array `refundStatusData` in the schema that will hold all the changes in status the RMA went through. The `status` field has to be sync'ed with it to represent always the latest one in the array. It will be useful having a validation in the process to update the status (a request cannot be processed if the status sent is previous to the current one).

Reply:
what are the possible statuses?

Reply:
Those:
``` 
"new",
"processing",
"pickedUpFromClient",
"pendingVerification",
"packageVerified",
"amountRefunded",
"denied"
```